public/app/themes/justice/error-pages/500.html

```diff
@@ -437,17 +437,17 @@
 
         </div>
     </header>
-
-    <!-- 403 HMTL error page starts here -->
-
-    <div class="u-wrapper l-main t-page-error">
-        <section class="c-rich-text-block c-page-error">
-            <h1 class="o-title o-title--page" style="font-size: 3rem; font-weight: 700;">Cannot connect to server</h1>
-            <p><strong>Server connection error - 500</strong></p>
-            <p>There is a problem connecting to the server. You could try to refresh your page several times.</p>
-            <p>If the problem persist please be patient, we are aware of the issue.</p>
-        </section>
-    </div>
+    <!-- 500 HTML error page starts here -->
+    <main>
+        <div class="u-wrapper l-main t-page-error">
+            <div class="c-rich-text-block c-page-error">
+                <h1 class="o-title o-title--page" style="font-size: 3rem; font-weight: 700;">Cannot connect to server</h1>
```

Comment:
If this is 2rem, it will match the 401 error page.

The difference can be seen: 

- http://justice.docker/app/themes/justice/error-pages/401.html
- http://justice.docker/app/themes/justice/error-pages/500.html